src/gui/optionsdialog.ui

```diff
@@ -459,6 +459,22 @@
               </layout>
              </widget>
             </item>
+            <item>
+             <widget class="QGroupBox" name="boxTorrentContent">
+              <property name="title">
+               <string>Torrent Content List</string>
```

Comment:
I would suggest "Torrent Content View". Although it is not really a "view" class but I think it is clearer for the average user to understand what it is referring to.